_guides/OMOP.md

```diff
@@ -0,0 +1,51 @@
+---
+title: OMOP
+shorthand: OMOP
+---
+
+
+#### What is it?
+OMOP is a common data model designed for standardising and analysing healthcare data from different sources for observational research. The OMOP Common Data model enables the structured examination of diverse observational databases. This methodology involves converting data from these databases into a unified format (data model) and consistent representation (terminologies, vocabularies, coding schemes). Subsequently, researchers can conduct systematic analysis using a set of standardised analytic routines specifically designed for the common format.
+
+#### Importance of Data Standardisaiton
+Data standardisation plays a crucial role in establishing a unified format of data. Allowing for collaborative research, enabling large scale analysis, and promoting the sharing of advanced tools. The importance of this arises due to significant variations in healthcare data across different organisations. Data is collected for various purposes consequently, it can be stored in diverse formats, using different systems and information models. Despite the increasing use of standard terminologies in healthcare, identical concepts can be represented differently at different organisations.  
```

Comment:
```suggestion
Data standardisation plays a crucial role in establishing a unified format of data, which allows collaborative research, enables large scale analysis, and promotes the sharing of advanced tools. The importance of this arises due to significant variations in healthcare data across different organisations. As data is collected for various purposes, it can be stored in diverse formats, using different systems and information models. Despite the increasing use of standard terminologies in healthcare, identical concepts can be represented differently at different organisations.  
```